aos_cd_jobs/updater_test.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+from unittest import TestCase, main
+from mock import Mock, call, patch
+
+from aos_cd_jobs.updater import create_branch, create_remote_branch, get_branch_by_name, list_jobs, update_branches
+
+class TestUpdateMethods(TestCase):
+    def test_list_jobs(self):
+        walker = (
+            ('dir/job0', None, ['Jenkinsfile']),
+            ('dir/notajob', None, ['']),
+            ('dir/job1', None, ['Jenkinsfile']),
+        )
+        self.assertEqual(tuple(list_jobs('dir', walker)), ('job0', 'job1'))
+
+    def test_get_branch_by_name(self):
+        branches = (Mock(), Mock())
+        branches[0].name = 'branch0'
+        branches[1].name = 'branch1'
+        self.assertEqual(get_branch_by_name(branches, 'branch0'), branches[0])
+        self.assertEqual(get_branch_by_name(branches, 'branch1'), branches[1])
+        self.assertIsNone(get_branch_by_name(branches, 'branch2'))
+
+    @patch('aos_cd_jobs.updater.rename')
+    @patch('aos_cd_jobs.updater.rmtree')
+    def test_create_branch(self, rmtree_mock, rename_mock):
```

Comment:
When you write more targeted functions like this one, it may make sense to write specific implementation tests (in the case of behavior verification, as is necessary here). Those types of tests are less appropriate for functions with larger scopes.